tests/bin/utils.ts

```diff
@@ -72,69 +73,97 @@ async function pkStdio(
   // (if not defined in the env) to ensure no attempted connections. A regular
   // PolykeyAgent is expected to initially connect to the mainnet seed nodes
   env['PK_SEED_NODES'] = env['PK_SEED_NODES'] ?? '';
-  // Parse the arguments of process.stdout.write and process.stderr.write
-  const parseArgs = (args) => {
-    const data = args[0];
-    if (typeof data === 'string') {
-      return data;
-    } else {
-      let encoding: BufferEncoding = 'utf8';
-      if (typeof args[1] === 'string') {
-        encoding = args[1] as BufferEncoding;
+  if (global.testCmd != null) {
```

Comment:
This should be `testCommand` to match the `PK_TEST_COMMAND` environment variable.